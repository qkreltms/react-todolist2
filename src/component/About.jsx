import React from 'react';
import { FormattedMessage } from 'react-intl'

const About = () => {
    return ( 
        <div>
            <h3>
                <FormattedMessage id='about' />
            </h3>
        </div>
     );
}
 
export default About;
