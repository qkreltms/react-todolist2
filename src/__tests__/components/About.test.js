import React from 'react';
import About from '../../component/About.jsx';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('About', () => {
    test('renders without crashing', () => {
        const component = shallow(<About />)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
      })
})
