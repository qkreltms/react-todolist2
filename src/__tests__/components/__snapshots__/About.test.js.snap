// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders without crashing 1`] = `
<div>
  <h3>
    <FormattedMessage
      id="about"
      values={Object {}}
    />
    <Connect(RandomPicture) />
  </h3>
</div>
`;
